lib/lambda_deployment/lambda/deploy.rb

```diff
@@ -37,6 +38,16 @@ def update_function_code
         )
       end
 
+      def update_environment
+        @client.lambda_client.update_function_configuration(
+          function_name: @config.project,
+          kms_key_arn: '',
```

Comment:
@rbayerl do you think this is good enough? According to [the docs](http://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/Lambda/Types/FunctionConfiguration.html#kms_key_arn-instance_method), this should cause it to use the default KMS key, so the env vars will be encrypted at rest.

If we need a custom key, we'll need to provide the ARN to this gem somehow.

Reply:
worth a try at least

Reply:
Looks like this is not possible with the default KMS key. Probably something we should fetch from the config file and/or environment.